verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -103,7 +103,7 @@ export default class CasesController {
         try {
             const response = await axios.delete(
                 this.dataServerURL + '/api' + req.url,
-                req.body,
+                { data: req.body },
```

Comment:
also if really needed, should data be mentioned in the openapi spec ?

Reply:
No, in the final request it is sent as the actual body. Just a weird thing with axios.